osbs/core.py

```diff
@@ -420,19 +420,31 @@ def logs(self, build_id, follow=False, build_json=None, wait_if_missing=False):
             return response.iter_lines()
         return response.content
 
-    def list_builds(self, build_config_id=None, field_selector=None):
+    def list_builds(self, build_config_id=None, koji_task_id=None,
+                    field_selector=None):
```

Comment:
Slightly concerned about changing the order of arguments, but it looks like optional arguments are always specified. LGTM.